client/src/helpers/api.ts

```diff
@@ -359,19 +361,16 @@ const updateServiceRequestStatus = async (
     return ext;
   });
 
-  let notes: any[] = get(serviceRequest, 'note', []).filter((n) => n.text != null);
+  const notes: any[] = get(serviceRequest, 'note', [...DEFAULT_NOTES]).filter((n: Note) => n.text != null);
```

Comment:
1. could you, if possible, find a type vs `any`
2. avoid coercion: `filter((n: Note) => n.text != null); => filter((n: Note) => n.text); `